FEXCore/Source/Interface/Core/Interpreter/Fallbacks/F80Fallbacks.h

```diff
@@ -77,6 +78,11 @@ struct OpHandlers<IR::OP_F80CVTTO> {
   FEXCORE_PRESERVE_ALL_ATTR static VectorRegType handle8(uint16_t FCW, double src, FEXCore::Core::CpuStateFrame* Frame) {
     FEXCORE_PROFILE_INSTANT_INCREMENT(Frame->Thread, AccumulatedFloatFallbackCount, 1);
     ScopedSoftFloatState State {FCW, Frame};
+    FEX_CONFIG_OPT(ReducedPrecisionMode, X87REDUCEDPRECISION);
```

Comment:
Would be nice if these were looking up the config from `Frame->Thread->CTX->Config` or something. While these are relatively lightweight because of the optimization to no longer do string conversions for every look up, they're still a map search each time. Do a bit of pointer dereferencing instead.